message/injector.cpp

```diff
@@ -1581,6 +1581,13 @@ void Injector::insertMessages()
 
         // Now we insert the headers and bodies of every MIME bodypart.
 
+        Bodypart *bp;
+        if ( m->hasPGPsignedPart() ) {
+            EString pnr( "raw-pgp-signed" );
```

Comment:
Can you show me an example where a part is signed without being a child of a multipart/signed?